cargo-concordium/src/build.rs

```diff
@@ -58,7 +58,7 @@ pub fn build_contract(
     // This assignment is not actually unused. It is used via the custom_section which retains a
     // reference to this vector, which is why it has to be here. This is a bit ugly, but not as
     // ugly as alternatives.
-    let mut schema_bytes = Vec::new();
+    let schema_bytes;
```

Comment:
This is strange. Why do we have an immutable uninitialized variable?

Reply:
The usual pattern is to assign it from a block or similar.

Reply:
because as described in the comment above `CustomSection` holds a reference to `schema_bytes`.